rust-js/README.md

```diff
@@ -0,0 +1,39 @@
+# Rust to WebAssembly in web and Node.js
+
+This directory contains examples of compiling Rust code to WebAssembly and using it on the web and in a Node.js application.
+The accompanying documentation can be found in the [Compiling from Rust to WebAssembly](https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_Wasm) article on MDN.
+
+## Running the examples
+
+This document assumes you are following the instructions in the MDN article which covers prerequisites and how to install the tools you need to build the example.
+
+If you already have Rust, `wasm-pack` and Node.js installed and you would like to skip the setup steps, the following two sections have instructions on how to run examples that have already been compiled, focusing only on the differences between the two.
```

Comment:
```suggestion
If you've already installed Rust, `wasm-pack`, and Node.js and wish to skip the setup steps, the next two sections provide instructions on how to run pre-compiled examples, focusing only on the differences between the two.
```
Not sure what "differences between the two" means. Should this be "the next two sections provide instructions on how to run pre-compiled examples, highlighting only the differences."?

Reply:
Good catch, updated your suggestion for clarity